stac-api/src/main/java/io/arlas/server/stac/api/StacSearchRESTService.java

```diff
@@ -71,19 +71,21 @@ public StacSearchRESTService(STACConfiguration configuration,
     @Path("/search")
     @GET
     @Produces({ "application/geo+json", MediaType.TEXT_HTML, MediaType.APPLICATION_JSON })
-    @ApiOperation(
-            value = "Search STAC items with simple filtering.",
-            notes = "Retrieve Items matching filters. Intended as a shorthand API for simple queries.")
+    @Operation(
+            summary = "Search STAC items with simple filtering.",
+            description = "Retrieve Items matching filters. Intended as a shorthand API for simple queries.")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "A feature collection.", response = StacFeatureCollection.class,
-                    responseContainer = "StacFeatureCollection"),
-            @ApiResponse(code = 400, message = "Invalid query parameter.", response = Error.class),
-            @ApiResponse(code = 500, message = "Arlas Server Error.", response = Error.class)})
+            @ApiResponse(responseCode = "200", description = "A feature collection.",
+                    content = @Content(array = @ArraySchema(schema = @Schema(implementation = StacFeatureCollection.class)))),
```

Comment:
Returns StacFeatureCollection.class (not array)

Reply:
ok